R/generateCI.R

```diff
@@ -141,7 +141,7 @@ generateCI <- function(stimuli, responses, baseimage, rdata, participants=NA,
   }
 
   # If "participants" argument is not given, compute one CI based on all data
-  if (all(is.na(participants))) {
+  if (all(is.na(participants))|length(unique(as.numeric(factor(participants)==1)))) {
```

Comment:
Can't we just use `length(unique(participants))` for this instead of first converting to factor and then numeric?